src/components/Navigation/Navigation.scss

```diff
@@ -208,14 +208,14 @@ $disabled-fade: 0.6;
     @include recolor-icon(
```

Comment:
Is this function used a bunch? I would assume it's a filter related function. If it is we could move it to this file.